tools/common_lib/src/conv.h

```diff
@@ -1506,7 +1506,7 @@ class ConvolutionCmDispatcher : public ConvolutionBaseDispatcher
                     const std::uint32_t ic_chunks_per_hw_thread = 8;
                     const std::uint32_t exec_size = 8;
                     const std::uint32_t dpas_depth = 8;
-                    const std::uint32_t out_dt_size = get_data_type_bytes_width(output_dt);
+                    const std::uint32_t out_dt_size = (std::uint8_t)get_data_type_bytes_width(output_dt);
```

Comment:
Variable is uint32_t, but cast to uint8_t was added, why not make these data types the same?

Reply:
Changed